livesync/indico_livesync/handler.py

```diff
@@ -88,12 +79,17 @@ def connect_signals(plugin):
     plugin.connect(signals.attachments.attachment_updated, _attachment_changed)
 
 
-def _moved(obj, old_parent, new_parent, **kwargs):
+def _moved_event(obj, old_category, **kwargs):
```

Comment:
I know it's a handler, but I think it's generally better to name the functions after what they do and not after what calls them.